lib/env-util.js

```diff
@@ -48,35 +49,41 @@ gutil.env.filterFiles = function (filePaths) {
 
 
 module.exports = function () {
-  if (is.object(sdkConf.ng) && is.not.empty(sdkConf.ng)) {
-    gutil.env.ng = {
-      module: sdkConf.ng.module || 'li',
-      textProperties: getSourcePaths(sdkConf.ng.addTextProperties || [])
-        .concat(['src/directives', 'src/services']),
-      moduleDependencyMap: sdkConf.ng.moduleDependencies || {},
-      moduleDependencies: getSourcePaths(
-        (function () {
-          var deps = [];
-          Object.keys(sdkConf.ng.moduleDependencies || {}).forEach(function (key) {
-            sdkConf.ng.moduleDependencies[key].forEach(function (dep) {
-              if (deps.indexOf(dep) === -1) {
-                deps.push(dep);
-              }
-            });
-          });
-          return deps;
-        })()
-      ),
-      gitReposForVersion: getSourcePaths(sdkConf.ng.gitReposForVersion || [])
-    };
-  } else {
-    gutil.env.ng = false;
-  }
-  gutil.env.gitStatusVersion = sdkConf.gitStatusVersion || false;
-  gutil.env.serverConfig = sdkConf.serverConfig;
-  gutil.env.bowerComponentsPath = BOWER_COMPONENTS_PATH;
-  gutil.env.watchResIgnore = sdkConf.watchResIgnore || [];
-  gutil.env.verifyPlugin = sdkConf.hasOwnProperty('verifyPlugin') ? sdkConf.verifyPlugin : true;
+    if (is.object(sdkConf.ng) && is.not.empty(sdkConf.ng)) {
```

Comment:
Indent here should be three spaces, not two. I suspect this is why the whole block appears to have changed (in the diff view) even though it didn't. Please put back to two spaces.